pkg/client/apiutil/apimachinery.go

```diff
@@ -27,10 +27,19 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/runtime/serializer"
 	"k8s.io/client-go/discovery"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/restmapper"
 )
 
+var (
+	localScheme = runtime.NewScheme()
+)
+
+func init() {
+	_ = clientgoscheme.AddToScheme(localScheme)
```

Comment:
this should not be using some sort of magic private scheme -- we should be using the user's passed-in scheme.  That way, if the user also defines protobuf support for their types (e.g. aggregated api servers, if we get proto support in the future for CRDs, etc) it'll just work.

Reply:
Done